server/base/src/main/java/org/apache/accumulo/server/master/balancer/GroupBalancer.java

```diff
@@ -70,15 +70,15 @@
 
 public abstract class GroupBalancer extends TabletBalancer {
 
-  private final String tableId;
+  private final Table.ID tableId;
   private long lastRun = 0;
 
   /**
    * @return A function that groups tablets into named groups.
    */
   protected abstract Function<KeyExtent,String> getPartitioner();
 
-  public GroupBalancer(String tableId) {
+  public GroupBalancer(Table.ID tableId) {
```

Comment:
This isn't strictly public API, but the balancer is one of those internal "points of code injection", for users to change Accumulo's default behavior. If API change can be avoided, that is preferred here.